docs/neopixel.rst

```diff
@@ -0,0 +1,69 @@
+Neopixel
+****
+
+.. py:module:: neopixel
+
+The ``neopixel`` module lets you use Neopixel (WS2812) individually addressable RGB LED strips with the Microbit.    
+Note to use the ``neopixel`` module, you need to import it separately with::
+
+    import neopixel
+
+.. warning::
+    
+    Do not use the 3v connector on the Microbit to power any more than 8 Neopixels at a time.
```

Comment:
Well, for Neopixels the max figure if you put them full white is 60ma/pixel. So for 8 pixels, that is 480ma. For 30 pixels that is 1.8 amps...
Of course, the more normal figure used is in general will draw 20ma/pixel, but even that at 30 pixels is 600ma!
The 3v3 pin is going through the regular, am assuming 600ma wouldn't end well...

We have to keep in mind with kids, chances are they will at some point turn them all on full white though by mistake, in turn blowing their microbit if using many.